src/langchain_google_alloydb_pg/async_vectorstore.py

```diff
@@ -766,6 +766,18 @@ def add_documents(
             "Sync methods are not implemented for AsyncAlloyDBVectorStore. Use AlloyDBVectorStore interface instead."
         )
 
+    def add_embeddings(
```

Comment:
I do like how this makes it clear to the user this isn't support, but since it's not a part of the interface let's remove this change.